book/docs/classes/OpusPool/Methods/getMaxUnstakeForUserForVault.md

```diff
@@ -0,0 +1,21 @@
+### getMaxUnstakeForUserForVault
+
+▸ **getMaxUnstakeForUserForVault**(`vault`): `Promise`\<`bigint`\>
+
+Retrieves the maximum amount of ETH that can be unstaked
+
+#### Parameters
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `vault` | \`0x$\{string}\` | A vault address |
+
+#### Returns
+
+`Promise`\<`bigint`\>
+
+A promise that resolves to the max amount of ETH that can be unstaked
+
+**`See`**
```

Comment:
I wasn't able to see this section, from `See` down, for any pf the methods in the [ui](https://chorus-one.gitbook.io/opus-pool-sdk-1.0/api-reference/opuspool). Worth checking if there is an error when publishing.

Reply:
It goes to the main gitbook, so once the PR is merged, the see would lead to the same page